src/useSynapse.js

```diff
@@ -0,0 +1,20 @@
+import React, { useContext, useState, useEffect } from 'react';
+import Context from './Context';
+import shallowEqual from './shallow-equal';
+
+export default function useSynapse(selector) {
+  const synapse = useContext(Context);
+  const select = () => selector(synapse.stores, synapse.state);
```

Comment:
do we need to expose both `synapse.stores` and `synapse.state`?

Reply:
nah